### DISCUSSION TITLE
RMarkdown Chunk Output directly below or in preview

### DISCUSSION THREAD

**Discussion by Xelamoon** [REditorSupport/vscode-R#1275] [Q&A] [answered]
Is it possible to get the output of a chunk directly below like in RStudio or Jupyter Notebooks in VS Code?

I haven't found an option. It was referenced or suggested here:
https://github.com/REditorSupport/vscode-R/issues/103#issuecomment-567212113

I know that live preview works for immediate changes but I sometimes would like to use the RMD File like a Notebook where I basically do not care about the actual HTML file only about a few chunks and what is in them.

Thanks!

**Comment by renkun-ken** [comment, marked as answer]
You should probably use a Jupyter Notebook with R kernel installed in this case.

**Comment by Xelamoon** [comment]
Thanks for the idea and answer! Also your work is much appreciated.
